etc/config.xml

```diff
@@ -8,17 +8,20 @@
             <integration>
                 <enable>1</enable>
             </integration>
+        </extend>
+        <extend_plans>
```

Comment:
fixing a bug - none of these defaults were being applied because the system.xml structure changed but this didn't change along with it